protocols/gossipsub/src/behaviour/tests.rs

```diff
@@ -105,6 +100,8 @@ where
                 },
                 i < self.outbound,
                 i < self.explicit,
+                Multiaddr::empty(),
+                self.peer_kind.clone().or(Some(PeerKind::Gossipsubv1_1)),
```

Comment:
```suggestion
                self.peer_kind.clone().unwrap_or(PeerKind::Gossipsubv1_1),
```

Reply:
hmm `add_peer_with_addr_and_kind()` seems to expect `Option<PeerKind>` this won't be necessary right?

Reply:
Ah woops, you're right, I missed that it must be an `Option.`